src/test/java/domain/LottoStatisticsTest.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoStatisticsTest {
+
+    @Test
+    void 정상당첨테스트() {
+        List<Lotto> lottoList = this.createLotto();
+        LottoStatistics lottoStatistics = new LottoStatistics(lottoList);
+
+        assertThat(lottoStatistics.getCountByRank(Rank.FIRST)).isEqualTo(1);
+        assertThat(lottoStatistics.getCountByRank(Rank.SECOND)).isEqualTo(1);
+        assertThat(lottoStatistics.getCountByRank(Rank.THIRD)).isEqualTo(0);
+        assertThat(lottoStatistics.getCountByRank(Rank.FOURTH)).isEqualTo(1);
+        assertThat(lottoStatistics.getCountByRank(Rank.FIFTH)).isEqualTo(1);
+        assertThat(lottoStatistics.getCountByRank(Rank.MISS)).isEqualTo(0);
```

Comment:
```suggestion
        List<Lotto> lottoList = this.createLotto();
        LottoStatistics lottoStatistics = new LottoStatistics(lottoList);

        assertThat(lottoStatistics.getCountByRank(Rank.FIRST)).isEqualTo(1);
        assertThat(lottoStatistics.getCountByRank(Rank.SECOND)).isEqualTo(1);
        assertThat(lottoStatistics.getCountByRank(Rank.THIRD)).isEqualTo(0);
        assertThat(lottoStatistics.getCountByRank(Rank.FOURTH)).isEqualTo(1);
        assertThat(lottoStatistics.getCountByRank(Rank.FIFTH)).isEqualTo(1);
        assertThat(lottoStatistics.getCountByRank(Rank.MISS)).isEqualTo(0);
```

>[테스트 픽스처 올바르게 사용하기](https://jojoldu.tistory.com/611)
> 이 부분을 읽고 보고 수정하려고하였는데,, 어떤부분이 적절하지 못한지 설명해주실수 있으실까요?
> 팩토리 메소드를 만들때, 인자를 주어야하는 부분인지,, 궁금합니다.

적절하지 못하여 코멘트 남긴것은 결코아니었습니다 ㅎ

Reply:
조금 고민되는것은,

1. 반환 객체 상태 불명확
   - createLotto 메소드가 생성하는 Lotto 객체들이 실제 어떤 상태인지 한눈에 파악하기 어려움.

2. 테스트 의존성 문제
   - 여러 테스트가 createLotto 메소드의 반환 상태에 강하게 의존하고 있음.
   - 만약 createLotto 내부 로직 또는 Lotto 객체의 상태가 변경되면, 일부 테스트는 실패하고 다른 테스트는 성공하는 불일치 현상이 발생할 가능성이 있음.

Reply:
@wooobo 
말씀해주신 방향으로 createLotto 사용자 입장에서 어떤 Lotto를 만드는지 아는 방향이 좋은것 같습니다.